### PR TITLE
TFT Feather template code and license updates.

### PR DESCRIPTION
**Pull request by kattni**
Code for the rest of the TFT Feather Template pages. Also updated licenses on all the CircuitPython_Templates/ files to MIT.

### CODE REVIEW

**Review by ladyada — APPROVED** [review]

**Comment by kattni** [comment]
Anne - please don't merge this (if you manage to get CI working), as Limor asked for a few changes that I still need to make. Thanks!

**Comment by TheKitty** [comment]
Got it @kattni

**Comment by kattni** [comment]
The template examples added in the https://github.com/adafruit/Adafruit_Learning_System_Guides/pull/2093/commits/a616e6c91e258739a3187df5939293eec7a84fdc commit will replace the files deleted in the https://github.com/adafruit/Adafruit_Learning_System_Guides/pull/2093/commits/fe33ebc503195663ccbff3cca6962dc1fa982103 commit.

**Pull request merged**
